docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -26,22 +26,20 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
   
 |Option|Meaning|  
 |------------|-------------|  
-|default|The compiler accepts all valid language syntax from the latest major version that it can support.|
-|ISO-1|The compiler accepts only syntax that is included in ISO/IEC 23270:2003 C# (1.0/1.2) <sup id="TISO1">[ISO1](#FISO1)</sup>|  
-|ISO-2|The compiler accepts only syntax that is included in ISO/IEC 23270:2006 C# (2.0) <sup id="TISO2">[ISO2](#FISO2)</sup>|
-|3|The compiler accepts only syntax that is included in C# 3.0 or lower <sup id="TCS3">[CS3](#FCS3)</sup>|
-|4|The compiler accepts only syntax that is included in C# 4.0 or lower <sup id="TCS4">[CS4](#FCS4)</sup>|
-|5|The compiler accepts only syntax that is included in C# 5.0 or lower <sup id="TCS5">[CS5](#FCS5)</sup>|
-|6|The compiler accepts only syntax that is included in C# 6.0 or lower <sup id="TCS6">[CS6](#FCS6)</sup>|
-|7|The compiler accepts only syntax that is included in C# 7.0 or lower <sup id="TCS7">[CS7](#FCS7)</sup>|
-|7.1|The compiler accepts only syntax that is included in C# 7.1 or lower <sup id="TCS71">[CS71](#FCS71)</sup>|
-|7.2|The compiler accepts only syntax that is included in C# 7.2 or lower <sup id="TCS72">[CS72](#FCS72)</sup>|
+|Preview|The compiler accepts all valid language syntax from the latest preview version that it can support.|
+|latest|The compiler accepts all valid language syntax from the latest version (including minor releases) that it can support.|
+|latestMajor|The compiler accepts all valid language syntax from the latest major version that it can support.|
```

Comment:
In the blog post, option names start with capital: `Latest` and `LatestMajor`. Throughout the changes in this PR both of them start with the small letter, though.

Reply:
That's by design. The element values are case-insensitive. Do you think that adds or detracts from understanding?

Reply:
> Do you think that adds or detracts from understanding?

As they are case-insensitive, it's more nit-picking then. Should then `Preview` be changed to `preview` for consistency?